src/app/components/bitcoin-contract-entry-point/bitcoin-contract-entry-point.tsx

```diff
@@ -42,14 +43,15 @@ export function BitcoinContractEntryPoint({ btcAddress }: BitcoinContractEntryPo
   }
 
   return (
-    <BitcoinContractEntryPointLayout
-      isLoading={isLoading}
-      balance={bitcoinContractSum}
-      caption={bitcoinContractSum.symbol}
-      icon={<Box as={BitcoinContractIcon} />}
-      usdBalance={i18nFormatCurrency(calculateFiatValue(bitcoinContractSum))}
-      cursor={'pointer'}
-      onClick={onClick}
-    />
+    !bitcoinContractSum.amount.isZero() && (
```

Comment:
The Bitcoin Contract Entry Point is now conditionally rendered; if the user has no locked Bitcoin, the component is not shown.